plugins/notifications-backend/src/service/db.ts

```diff
@@ -0,0 +1,87 @@
+import { Config } from '@backstage/config';
+
+import knex, { Knex } from 'knex';
+import merge from 'lodash/merge';
+
+// initDB
+// creates DB client and tables
+export async function initDB(dbConfig: Config): Promise<Knex<any, any>> {
+  // create db client
+  const knexConfig = merge(
+    {},
+    { connection: { database: 'backstage_plugin_notifications' } },
+    dbConfig.get(),
+    dbConfig.getOptional('knexConfig'),
+  );
+  const dbClient = knex(knexConfig);
+
+  // create tables
+  if (!(await dbClient.schema.hasTable('messages'))) {
+    await dbClient.schema.createTable('messages', table => {
+      table.uuid('id', { primaryKey: true }).defaultTo(dbClient.fn.uuid());
+      table.string('origin').notNullable();
+      table.timestamp('created').defaultTo(dbClient.fn.now()).index();
+      table.string('title').notNullable();
+      table.text('message');
```

Comment:
I would like to see also `status` field for the messages. When the user fetches the messages, the system could automatically mark those as `read` and indicate that later in the UI which are new and which are already read messages. What do you think?